src/components/FilterForm/index.jsx

```diff
@@ -0,0 +1,96 @@
+import { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import MenuItem from '@material-ui/core/MenuItem';
+import Menu from '@material-ui/core/Menu';
+import { withStyles } from '@material-ui/core/styles';
+import { parse, stringify } from 'qs';
+import AccountIcon from 'mdi-react/AccountIcon';
+import { ASSIGNEE } from '../../utils/constants';
+
+const assigneeList = Object.values(ASSIGNEE);
+
+@withRouter
+@withStyles(theme => ({
+  root: {
+    margin: `0px ${3 * theme.spacing.unit}px`,
+    maxWidth: 360,
+  },
+}))
+export default class FilterMenu extends Component {
```

Comment:
The directory name is `FilterForm` but the component name is `FilterMenu`. They should be the same.